Soccar [Library]/src/nl/soccar/library/enumeration/Duration.java

```diff
@@ -7,8 +7,31 @@
  */
 public enum Duration {
     
-    MINUTES_3,
-    MINUTES_5,
-    MINUTES_10
+    MINUTES_3(0),
+    MINUTES_5(1),
+    MINUTES_10(2);
+    
+    private final int id;
+
+    private Duration(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static Duration parse(int id) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)